gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=6.0.0
+VERSION_NAME=6.0.1-SNAPSHOT
```

Comment:
okay to have `-SNAPSHOT` here in master?

Reply:
yes, it should. I will figure out why the release script did not change it.